controller/volume_controller.go

```diff
@@ -1566,11 +1915,26 @@ func (vc *VolumeController) getReplenishReplicasCount(v *longhorn.Volume, rs map
 			usableCount++
 		}
 	}
-
-	if v.Spec.NumberOfReplicas < usableCount {
+	rCount := v.Spec.NumberOfReplicas
```

Comment:
Just have 3 conditions, suggest use `v.Spec.NumberOfReplicas` directly instead of creating another variable.

Reply:
Addressed.